pkg/query/api/v1.go

```diff
@@ -269,6 +270,21 @@ func (api *API) parseReplicaLabelsParam(r *http.Request) (replicaLabels []string
 	return replicaLabels, nil
 }
 
+func (api *API) parseStoreAllowListParam(r *http.Request) (storeAllowList []string, _ *ApiError) {
```

Comment:
It's fine to treat `Allowlist` as one word so no need to do camel case for that.

But again it would be nice to put explicit `Address` there (: